lib/anyway/settings.rb

```diff
@@ -65,6 +65,10 @@ def default_config_path=(val)
       def future
         @future ||= Future.new
       end
+
+      def current_environment
+        ENV["RAILS_ENV"].present? ? ::Rails.env : nil
```

Comment:
All Rails specific logic should go under `anyway/rails` (as it is now).
We cannot rely on `RAILS_ENV` presence to make sure that `::Rails` constant is defined. Some environments always have this variable set by may invoke non-Rails Ruby code (e.g., some tasks within a Rails app Docker container).